src/components/Signup/ExistingPodForm.jsx

```diff
@@ -4,13 +4,17 @@ import React, { useState } from 'react';
 import { useSession } from '@hooks';
 // Material UI Imports
 import Button from '@mui/material/Button';
-import Card from '@mui/material/Card';
-import CardHeader from '@mui/material/CardHeader';
+import Box from '@mui/material/Box';
 import TextField from '@mui/material/TextField';
 
 /* Styles */
 const formRowStyle = {
-  margin: '20px 0'
+  m: 1
```

Comment:
Just to make sure, is this the same thing as margin? So `m: 1` is equivalent to `margin: 1rem` or something in that light.

Reply:
> Spacing
> The spacing properties margin, padding, and the corresponding longhand properties multiply the values they receive by the theme.spacing value (the default for the value is 8px):

https://mui.com/system/getting-started/the-sx-prop/#spacing